.dotnet/src/Generated/Models/ChatCompletionOptions.Serialization.cs

```diff
@@ -131,6 +131,11 @@ void IJsonModel<ChatCompletionOptions>.Write(Utf8JsonWriter writer, ModelReaderW
                     writer.WriteNull("seed");
                 }
             }
+            if (Optional.IsDefined(ServiceTier))
+            {
+                writer.WritePropertyName("service_tier"u8);
+                writer.WriteStringValue(ServiceTier.Value.ToString());
+            }
```

Comment:
This one is also nullable. Just double-checking the serialization in the Write and Deserialize methods.